### ISSUE TITLE
linux is missing 

### ISSUE THREAD

**Issue by Pegasus20x** [sebanc/brunch#1518]
i have brother DCP-T300 printer ,but it seems like it's not supported on chrome os. i have the linux driver for the printer. will it fix the issue?
the problem is i can't find linux installer anymore. i tried powerwash but still missing

**Comment by tejasraman** [comment]
Enable virtualization in BIOS
What recovery image did you use?

**Comment by tejasraman** [comment]
Ohhh. I say your other post - you have a 4th gen I7.
Use Rammus image 100 with Brunch 100

**Comment by Pegasus20x** [comment]
> Enable virtualization in BIOS What recovery image did you use?

It's enabled

**Comment by Pegasus20x** [comment]
> Ohhh. I say your other post - you have a 4th gen I7. Use Rammus image 100 with Brunch 100

I use rammus 100 with brunch 100

**Comment by tejasraman** [comment]
Ok... I really have no idea about this. Take a screenshot of settings pls?

**Comment by Pegasus20x** [comment]
> Ok... I really have no idea about this. Take a screenshot of settings pls?
![Screenshot 2022-04-16 12 10 30 AM](https://user-images.githubusercontent.com/87944574/163648461-5636d1b6-4ebf-4330-ad5b-a4526c4a3497.png)

**Comment by tejasraman** [comment]
Oh... thanks for the screenshot! Did you check advanced - developers in settings?

Is this a managed account? I had a managed account and it didn't support Linux - it is an option admins can enable and in Family Link(if that is what you are using) Linux cannot be enabled.

Also(have a **talk** with your admin if it is managed - that is the problem) is this in an NTFS partition? (if so, 😢) and switch to EXT4 with a Linux live USB (if you don't have Linux download a Linux Mint ISO and see [here](https://trendoceans.com/how-to-create-a-bootable-usb-using-rufus-for-linux-distributions/) to make a USB)

**Comment by Pegasus20x** [comment]
> Oh... thanks for the screenshot! Did you check advanced - developers in settings?
> 
> Is this a managed account? I had a managed account and it didn't support Linux - it is an option admins can enable and in Family Link(if that is what you are using) Linux cannot be enabled.
> 
> Also(have a **talk** with your admin if it is managed - that is the problem) is this in an NTFS partition? (if so, 😢) and switch to EXT4 with a Linux live USB (if you don't have Linux download a Linux Mint ISO and see [here](https://trendoceans.com/how-to-create-a-bootable-usb-using-rufus-for-linux-distributions/) to make a USB)
> 
> Also, I notice you enabled the light/dark theme flag :)

i am the admin of my account . i installed chrome os last year and i had the linux . i do have the partition in NTFS.

**Comment by tejasraman** [comment]
weird... If this started recently, I cannot help you past this. I have seen a few problems with people using NTFS, so maybe put it onto ETXT4?

**Comment by tejasraman** [comment]
and you will have to print from linux for the driver to work.

**Comment by tejasraman** [comment]
@sebanc is Brunch 100 broken?

**Comment by Pegasus20x** [comment]
> and you will have to print from linux for the driver to work.

so you're saying that even if i managed to install linux and the linux driver , i still won't be able to print using chrome ,and i'll have to print using linux terminal?

**Comment by tejasraman** [comment]
No… You will have to download a Linux version of Chrome or Edge or Firefox and print from it
But you cannot print from the built in chrome unless you use crosh and types shell to access a system level Linux shell (there you can install the driver as system).
